jazzhands/__main__.py

```diff
@@ -196,45 +269,68 @@ def main(argv):
 
         if index_files['js'] and js_dir:
             jsx_registry_path = os.path.join(os.path.dirname(index_files['js']), 'jsx_registry.js')
-            subprocess.call(['python', 'manage.py', 'compilejsx'], stdout=open(jsx_registry_path, 'w'))
-            process_jsx(jsx_registry_path)
-            
+            try:
+                import django_jsx  # noqa: F401
+                subprocess.call(['python', 'manage.py', 'compilejsx', '-o', jsx_registry_path])
+                process_jsx(jsx_registry_path)
+            except ImportError:
```

Comment:
I'd prefer to move the two lines after the import into an "else" part. That way, if those two lines happen to raise an ImportError, it won't be ignored.